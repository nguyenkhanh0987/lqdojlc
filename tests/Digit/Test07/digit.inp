100
83754 87062
14122 18943
68740 77155
91550 98470
61146 85905
13600 88859
6529 60831
6807 76147
51895 76576
16780 56442
67725 83982
18613 22209
9109 10851
10179 98748
18760 91117
41905 79344
12335 29018
26102 34127
68703 90797
69198 86371
28006 70429
20855 84627
72610 90658
55406 83103
66917 90459
41249 71262
21747 33773
10498 42166
84596 99429
79068 91774
66087 73460
50128 90685
46153 81012
66149 80137
89656 92184
29592 91165
41365 48067
85092 96054
88141 97602
13941 76463
53673 86413
59078 82847
81019 82488
7416 66328
78192 91786
44766 74102
84467 85090
90459 93032
12490 90402
23828 24999
26544 62497
74626 82719
33490 75839
70343 83792
87375 98344
68193 69716
10118 61727
90063 93847
36702 87330
82508 87390
10117 58780
53172 89493
39006 86925
29138 93317
79027 81597
16035 39509
33419 83067
34139 64328
68864 84695
62055 99024
64492 81989
60377 92629
87877 92887
81539 84505
58845 69643
7261 48152
83957 98145
38978 71156
30511 38266
94154 97441
25536 70409
21725 29605
1931 89652
95634 97462
67709 89580
7537 83204
87878 91564
70247 87738
79359 96391
53831 70606
73787 75252
99796 99875
69215 93992
53350 86062
53701 77085
94344 95580
4315 75330
25298 27255
18322 35356
26866 98674
