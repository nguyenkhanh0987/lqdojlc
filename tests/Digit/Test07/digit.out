95
1165
379
156
1237
8067
8092
8826
1706
5442
845
558
367
8702
6993
3100
3100
1247
1053
712
4666
6243
852
1860
1300
2668
1776
5005
562
486
531
2433
2816
998
10
5040
651
353
274
7452
2078
1377
5
8058
571
2507
8
1
8326
145
4317
451
3982
518
358
251
7011
5
4102
148
6773
2347
3580
5208
140
4100
4395
3248
718
1700
1088
1545
198
30
875
6599
469
2985
933
88
5169
1475
10592
67
1178
8999
197
668
476
1476
54
0
981
2053
1657
17
9401
390
2591
5955
